config/locales/account_reset/es.yml

```
@@ -2,61 +2,59 @@
-      cta_html: Puede %{link_html} o cierra esta ventana si ya terminaste.
-      info_html: La cuenta para <strong>%{email}</strong> ha sido eliminada. Nosotros
-        enviamos una confirmación por correo electrónico de la eliminación de la
-        cuenta.
-      link_text: crea una cuenta nueva
-      title: Has eliminado tu cuenta
+      cta_html: Puede %{link_html} o cerrar esta ventana si ya terminó.
+      info_html: La cuenta de <strong>%{email}</strong> fue eliminada. Enviamos una
+        confirmación por correo electrónico de la eliminación de la cuenta.
+      link_text: crear una cuenta nueva
+      title: Eliminó su cuenta
     confirm_request:
-      close_window: Puede cerrar esta ventana si ha terminado.
+      close_window: Puede cerrar esta ventana si ya terminó.
       instructions_end: para comenzar el proceso de eliminación de cuenta. Siga las
         instrucciones en su correo electrónico para completar el proceso.
       instructions_start: Enviamos un correo electrónico a
       security_note: Como medida de seguridad, también enviamos un mensaje de texto a
-        su registro número de teléfono.
+        su número de teléfono registrado.
     delete_account:
-      are_you_sure: '¿Seguro que quieres eliminar tu cuenta?'
-      info: Eliminar su cuenta debe ser su último recurso si su cuenta está bloqueada.
-        No podrá recuperar ninguna información vinculada a su cuenta.
-        Notificaremos a las agencias a las que acceda con %{app_name} que ya no
-        tiene una cuenta. Cuando su cuenta sea eliminada podrá crear una nueva
-        usando la misma dirección de correo electrónico.
-      title: Eliminar tu cuenta debería ser tu último recurso
+      are_you_sure: '¿Está seguro de que desea eliminar su cuenta?'
+      info: Eliminar su cuenta debe ser su último recurso si el acceso a su cuenta
+        está bloqueado. No podrá recuperar ninguna información vinculada a su
+        cuenta. Notificaremos a las agencias a las que accede con %{app_name}
+        que usted ya no tiene una cuenta. Cuando elimine su cuenta, puede crear
+        una nueva usando la misma dirección de correo electrónico.
+      title: Eliminar su cuenta debe ser su último recurso.
     pending:
-      cancel_request: Cancelar petición
-      cancelled: Hemos cancelado su solicitud para eliminar su cuenta.
-      confirm: Si cancela ahora, debe crear una nueva solicitud y esperar otras
-        %{interval} para eliminar su cuenta.
-      header: Solicitaste eliminar tu cuenta
-      wait_html: Hay un período de espera de %{hours} horas para eliminar su cuenta.
-        En <strong>%{interval}</strong>, recibirá un correo electrónico con
-        instrucciones para completar la eliminación.
+      cancel_request: Cancelar solicitud
+      cancelled: Cancelamos su solicitud para eliminar su cuenta.
+      confirm: Si la cancela ahora, debe crear una nueva solicitud y esperar otras
+        24 horas para eliminar su cuenta.
```

```suggestion
        %{interval} horas para eliminar su cuenta.
```

The 24 hours is now interpolated from a variable so that if we update the timeframe the content matches it. We should be able to maintain the new translation and substitute the variable.